dashboard/test/ui/features/code_tools/google_blockly/modal_function_editor.feature

```diff
@@ -31,7 +31,7 @@ Scenario: Can edit a function
   # Open Sprites flyout
   And I press "blockly-d"
   # Drag new sprite block to top of function
-  And I drag block number 3 to offset "40, 100"
+  And I drag block number 2 to offset "40, 100"
```

Comment:
for my knowledge: why did this change?

Reply:
Oh! I captured this in the commit message, but didn't leave a comment sorry. When we find blocks with this step, we are only counting draggable blocks. Now that the definition block is immovable like it's supposed to be, it no longer counts as a draggable block.